package.json

```diff
@@ -4,27 +4,33 @@
   "engines": {
     "node": ">=10"
   },
+  "dependencies": {
+    "vue-bar-graph": "1.2.0"
```

Comment:
It's necessary for runtime?

Reply:
dependencies -> is client side javascript (even if it is compiled to other file later)
devDependencies -> development tools